assets/js/admin/meta-boxes-product.js

```diff
@@ -464,6 +464,14 @@ jQuery( function( $ ) {
 				$( '.product_attributes' ).html( response.data.html );
 				$( '.product_attributes' ).unblock();
 
+				// Make sure the dropdown is not disabled for empty value attributes.
+				var nr_elements = original_data.length / 6;
+				for ( i = 0; i < nr_elements; i++ ) {
+					if ( typeof( original_data ) != "undefined" && original_data[ i * 6 + 2 ].value === "" ) {
```

Comment:
We don't use double quotes in our JS files, this will trigger errors on grunt, you can test first with `grunt jshint`.